component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1658,4 +1669,140 @@ private long getUnlockTimeInMilliSeconds(AuthenticatedUser authenticatedUser) th
                     username, e);
         }
     }
+
+    /**
+     * Send event after generating SMS OTP.
```

Comment:
```suggestion
     * Trigger event after generating SMS OTP.
```

Reply:
Fixed in 48dfc684db60965170330a1fab99c186531f743e